web/firefoxcom.js

```diff
@@ -148,6 +148,32 @@ Preferences._readFromStorage = function (prefObj) {
   });
 };
 
+(function listenFindEvents() {
+  var events = [
+    'find',
+    'findagain',
+    'findhighlightallchange',
+    'findcasesensitivitychange'
+  ];
+  var handleEvent = function (evt) {
```

Comment:
I decided to change that only for new code and non-DOM handlers (this one is a copy'n'paste from and also DOM handlers).